project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
```

Comment:
We usually use `addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.0.0")`

Reply:
Oh. I just copied what was in the ivy repo.

Reply:
From what I can tell, sbt-dynver does not work well with this repo because the tags do not match the formatting scheme.